### PR TITLE
sleep

### PR DESCRIPTION
**Pull request by AliCl0ner**
added night sleep
rechare time + random timer
proxy error handling
and fixed some bugs

### CODE REVIEW

File: req.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
-telethon>=1.37.0
-requests>=2.28.1
-urllib3>=1.26.12
-python-dotenv
+telethon>=1.37.0
+requests>=2.28.1
+urllib3>=1.26.12
+python-dotenv
+pytz
```
Comment (dmitry-js) [inline req.txt:5]: why did you rename `requirements` to `req` and what's the point of the `pytz` lib?

**Comment by dmitry-js** [comment]
hi @AliCl0ner why did you close this?

**Comment by AliCl0ner** [comment]
Hi
I rewrite all the code again and removed fork
